runtime/autoload/man.vim

```diff
@@ -12,6 +12,11 @@ catch /E145:/
   " Ignore the error in restricted mode
 endtry
 
+try
```

Comment:
What’s the point of `try` here? It is transforming all errors inside into exceptions and not catching after that, not what you would need at startup. I would suggest just `silent!` the call and remove `try`/`endtry`.